processing/src/test/java/org/apache/druid/query/DoubleStorageTest.java

```diff
@@ -321,10 +322,10 @@ private static QueryableIndex buildIndex(String storeDoubleAsFloat) throws IOExc
         )
         .build();
 
-    final IncrementalIndex index = new IncrementalIndex.Builder()
```

Comment:
from here on forward same 5 lines changes repeat for different tests

Reply:
This is true for tests that do not test the incremental-index implementation. For such tests, we use the on-heap implementation because it is the most stable.
Tests that test the index itself, are now parametrized so they have more modifications other than these 5 lines.